Tests/SkopelosTests/SkopelosTests.swift

```diff
@@ -0,0 +1,11 @@
+    import XCTest
+    @testable import Skopelos
+
+    final class SkopelosTests: XCTestCase {
+        func testExample() {
+            // This is an example of a functional test case.
+            // Use XCTAssert and related functions to verify your tests produce the correct
+            // results.
+            XCTAssertEqual(Skopelos().text, "Hello, World!")
```

Comment:
Thanks for contributing @shrtlist 🙏
The rest looks ok, if you could only address this.

Reply:
You're welcome! Updated.